### ISSUE TITLE
S1144: Unused private members should not report on async Main method in Program.cs

### ISSUE THREAD

**Issue by alexeyzimarev** [SonarSource/sonar-csharp#1069]
In C# 7.1 it is possible to declare the `Program.Main` method `async`. Sonar C# reports a false positive on this:

Remove the unused private method 'Main'
```csharp
private static async Task Main()
{
    // some code
}
```

**Comment by Evangelink** [comment]
Could you let us know which version of SonarC# you are using? I remember having fixed this issue in #888.
